db/migrate/20210614130500_add_structured_gcse_columns_to_course_table.rb

```diff
@@ -0,0 +1,10 @@
+class AddStructuredGcseColumnsToCourseTable < ActiveRecord::Migration[6.1]
+  def change
+    add_column :course, :accept_pending_gcse, :boolean
```

Comment:
This is probably a follow up to this comment https://github.com/DFE-Digital/teacher-training-api/pull/1940#discussion_r648495323, these boolean fields should ideally default to false assuming the fields are opt-in (driven by the user to make a selection to turn it on)